HW03/vorobyov_egor/lib/predicates.rb

```diff
@@ -0,0 +1,5 @@
+module Predicates
+  def self.with_error(el)
+    el if el.include? 'error'
```

Comment:
We usually don't create module just for one line at all 😅 
```suggestion
  def self.with_error?(el)
    el.include? 'error'
```